sdk/hybridnetwork/Azure.ResourceManager.HybridNetwork/src/Generated/Models/DaemonSet.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+
+namespace Azure.ResourceManager.HybridNetwork.Models
+{
+    /// <summary> Helm DaemonSet status properties. </summary>
+    public partial class DaemonSet
+    {
+        /// <summary> Initializes a new instance of DaemonSet. </summary>
+        internal DaemonSet()
+        {
+        }
+
+        /// <summary> Initializes a new instance of DaemonSet. </summary>
+        /// <param name="name"> The name of the daemonSet. </param>
+        /// <param name="namespace"> The namespace of the daemonSet. </param>
+        /// <param name="desired"> Desired number of pods. </param>
+        /// <param name="current"> Current number of pods. </param>
+        /// <param name="ready"> Number of Ready pods. </param>
+        /// <param name="upToDate"> Number of  upto date pods. </param>
+        /// <param name="available"> Number of available pods. </param>
+        /// <param name="createdOn"> Creation Time of daemonSet. </param>
+        internal DaemonSet(string name, string @namespace, int? desired, int? current, int? ready, int? upToDate, int? available, DateTimeOffset? createdOn)
+        {
+            Name = name;
+            Namespace = @namespace;
+            Desired = desired;
+            Current = current;
+            Ready = ready;
+            UpToDate = upToDate;
+            Available = available;
+            CreatedOn = createdOn;
+        }
+
+        /// <summary> The name of the daemonSet. </summary>
+        public string Name { get; }
+        /// <summary> The namespace of the daemonSet. </summary>
+        public string Namespace { get; }
+        /// <summary> Desired number of pods. </summary>
+        public int? Desired { get; }
```

Comment:
The name is too vague. Suggest to rename it as `NumberOfDesired` or similar. Same as the properties below.